app/__init__.py

```diff
@@ -1,12 +1,12 @@
 from flask import Flask
-from .db import db, migrate
-from .routes.planet_routes import planets_bp
+from app.db import db, migrate
+from app.routes.planet_routes import planets_bp
```

Comment:
Noted these imports changed from relative imports --> absolute imports. Is this the recommended style?